integration/kubernetes/k8s-copy-file.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	pod_name="test-env"
+	get_pod_config_dir
+	file_name="file.txt"
+}
+
+@test "Copy file in a pod" {
+	# Create pod
+	kubectl create -f "${pod_config_dir}/pod-env.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# Create a file
+	echo "Hello" > "$file_name"
```

Comment:
`"Hello"` can be a variable too and use it later to check file's content

Reply:
@devimc changes applied